python/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+from hiro_client import __version__
+
+setup(
+    name="hiro_client",
+    version=__version__,
+    packages=find_packages(),
+
+    author="Wolfgang Hübner (arago GmbH)",
+    author_email="info@arago.co",
+    description="Hiro Client for Graph REST API of HIRO 7",
+    keywords="HIRO7 connector arago REST API",
+    url="https://github.com/arago/hiro-clients",
+    license="MIT",
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
+    ]
+)
```

Comment:
As I just found out that type specifications are not compatible with Python 3.5, this line should be included here:
`python_requires='>=3.7'`

See here: https://packaging.python.org/tutorials/packaging-projects/#creating-setup-py

Reply:
Added the python_requires :)